nrf_security/src/psa/psa_eits_zephyr_settings.c

```diff
@@ -10,13 +10,49 @@
 #include <sys/printk.h>
 #include <storage/flash_map.h>
 
+#include <nrf_cc3xx_platform.h>
+#include <nrf_cc3xx_platform_ctr_drbg.h>
+#include <nrf_cc3xx_platform_derived_key.h>
+
 #include "psa_eits_backend.h"
 
 LOG_MODULE_REGISTER(psa_eits_zephyr_settings,
 		    CONFIG_PSA_EITS_BACKEND_ZEPHYR_LOG_LEVEL);
 
+
+const uint8_t deriv_context[]="static_deriv_context";
+
 #define ROOT_KEY "EITS"
-#define MAX_PATH_LEN 32
+#define MAX_PATH_LEN 64
+#define DERIV_KEY_SIZE 128
+#define DERIV_KEY_SLOT  0
+#define ENC_TAG_SIZE 16
+#define ENC_IV_SIZE 16
+#define ENC_BUFF_SIZE 1024
+#define DERIV_LABEL_SIZE (sizeof(psa_storage_uid_t) + sizeof(deriv_context))
+#define ENC_ADD_DATA_SIZE (sizeof(psa_storage_uid_t)+ \
+                           ENC_IV_SIZE+ \
+                           sizeof(uint32_t)+ \
+                           sizeof(psa_storage_create_flags_t))
+
+
+static uint8_t encrypted_buffer[CONFIG_PSA_EITS_READ_BUFF_SIZE];
+static uint8_t decrypted_buffer[CONFIG_PSA_EITS_READ_BUFF_SIZE];
+static uint8_t deriv_label[DERIV_LABEL_SIZE];
+static uint8_t tag[ENC_TAG_SIZE];
+static uint8_t iv[ENC_IV_SIZE];
+static uint8_t add_data[ENC_ADD_DATA_SIZE];
+
+K_MUTEX_DEFINE(eits_mutex);
```

Comment:
Could a comment be added here to explicitly state exactly what this mutex needs to protect?